src/jit/lower.h

```diff
@@ -189,6 +190,7 @@ class Lowering : public Phase
     void TreeNodeInfoInitReturn(GenTree* tree);
     void TreeNodeInfoInitShiftRotate(GenTree* tree);
     void TreeNodeInfoInitCall(GenTreeCall* call);
+    void TreeNodeInfoInitCmp(GenTreePtr tree);
     void TreeNodeInfoInitStructArg(GenTreePtr structArg);
```

Comment:
I'm going to leave things as-is for now, but I wonder if we might as well move `LowerCompare` up to `DecomposeLongs`, which would resolve the naming conflict.